sections/introduction.tex

```diff
@@ -1,23 +1,32 @@
+A general challenge for search engines, no matter the environment, is how to acquire relevant search results from a given corpus. 
+While some engines use simple algorithms, such as finding all articles including a given search term and sorting by date, this may not always give useful results.
+Ranking articles can be done multiple ways, where PageRank is commonly used\cite{google_pagerank2006}.
+PageRank\cite{pagerank_1999} is a ranking algorithm, which is based on the random walk process.
+The principle is to let a number of surfers walk nodes in a graph, where each surfer takes one step in each iteration.
+After some steps the number of surfers in one node is the ranking of that given node.
+The edges in the graph can be based on many interactions, such as hyperlinks, similarities, genre, etc.
 Therefore we want to explore ways to improve search results while using information already available from a corpus.
-\todo[inline]{jumping between labels and search}
-Labeling documents in a corpus and using these labels to get relevant results could improve queries.
-A popular way to label documents is through the use of topic modeling. \todo[inline]{Overvej om der er et bedre ord en "label", da det måske kan være misvisende.}
-Topic modeling gives each document in a corpus a distribution of topics, which can be used for different applications.
+Documents in a corpus can contain relevant information, which could yield improvements for search engines.
+A popular way to extract this information is through the use of topic modeling.
+Topic modeling is method within machine learning and \gls{NLP} which builds a model, that can find abstract topics with a corpus of text.
+A topic model assigns a distribution of topics for each document in a corpus, which can be used for different applications.
 More specifically, we look into \gls{lda} since it is currently the most used topic model, and is applicable for many situations\cite{lda}.
-\todo[inline]{explain what topic modelling is and what it actually does}
-Topic modeling does not rank documents in a search, which is why an additional part is necessary.
-\todo[inline]{add two sentences to explain everything}
-We have chosen to explore a popular way to rank documents, which is by using PageRank.\todo[inline]{add half a sentence explaining the principle}
+\gls{lda} uses Dirichlet distributions to calculate the probability of a word belonging to a given topic and document containing a given topic.
+Topic modeling does not rank documents, which is why an additional part is necessary if we want to search the documents.
+Can topic information be used to improve search engines, is what we intend to investigate.
+We have chosen to explore a popular way to rank documents, which is by using PageRank.
 PageRank is an algorithm originally used for ranking web pages in search engines, but it can easily be extended to other types of data, such as news articles.
```

Comment:
Synes det lyder mystisk at snakke om pagerank igen som var det et nyt emne. Ved ikke hvad den gode løsning er, om man skal rykke den tidligere beskrivelse herned, eller bare prøve at lave en bedre introduktion til at snakke om pagerank igen.

Reply:
Jeg ved ikke lige hvordan jeg skal løse den

Reply:
Jeg prøver lige at kigge på det.

Reply:
Her er et forslag jeg synes gør det ok at nævne pagerank tidligere. Så lyder det mindre som om vi snakker om et helt nyt emne.
```suggestion
We have chosen to explore using PageRank along with topic information, since it is a popular way to rank documents.
PageRank was originally used for ranking web pages in search engines, but it can easily be extended to other types of data, such as news articles.
```